.travis/build-pkgs

```diff
@@ -65,11 +66,11 @@ done
 
 printf '\n----\n'
 if [ -n "$successful_pkgs" ]; then
-	print -s1 -c2 "Successfully build packages:$successful_pkgs\n"
+	print -s1 -c2 "Successfully build packages: $successful_pkgs\n"
 fi
 if [ -n "$failed_pkgs" ]; then
-	die "Failed to build packages:$failed_pkgs"
+	die "Failed to build packages: $failed_pkgs"
 
 elif [ -z "$successful_pkgs" ]; then
-	die 'No packages found to be build.'
+	warn 'No packages found to be built.'
```

Comment:
We don't want to fail if no packages have been found to require a build (see above).